src/b3d/io/utils.py

```diff
@@ -172,9 +172,6 @@ def video_input_from_mp4(
     downsize=1,
     reverse_color_channel=False,
 ):
-    if times is None:
```

Comment:
@mirkoklukas , `T` wasn't available, so we can't do this calculation here; we could pass `times` back OUT of `load_video_to_numpy`, but that's a bigger change.

Can you let us know if the below change, where we swap your `fps` for `info.fps`, is valid?

Reply:
I think adding `T = info.timesteps` before the None check should do the trick.

Reply:
Boom, fixed!